lib/src/page_list_viewport_gestures.dart

```diff
@@ -308,15 +341,28 @@ class DeprecatedPanAndScaleVelocityTracker {
 
   int _previousPointerCount = 0;
   int? _previousGestureEndTimeInMillis;
-  int? _previousGesturePointerCount;
 
   int? _currentGestureStartTimeInMillis;
   PanAndScaleGestureAction? _currentGestureStartAction;
   bool _isPossibleGestureContinuation = false;
 
+  // Variables needed to keep track of repeated input scroll acceleration
```

Comment:
This comment doesn't help me understand why we need these variables. What's a "repeated input scroll acceleration" and why do we need variables to track it?

"for for" -> "for"

"Consider the swipe for repeated input acceleration" - consider what about it? What are you trying to say?